content/beginner/080_scaling/cleanup.md

```diff
@@ -33,6 +33,8 @@ kubectl delete deployment php-apache
 
 kubectl delete pod load-generator
 
+kubectl delete deployment load-generator
```

Comment:
Can you point me to where the _load-generator_ deployment is created ? I don't see any creation of that resource :) I think you played around by yourself (Which is definitely encouraged!) but we can't assume other people will create the same resources...